src/modules/musig/main_impl.h

```diff
@@ -502,6 +539,26 @@ int secp256k1_musig_partial_sig_combine(const secp256k1_context* ctx, const secp
         secp256k1_scalar_add(&s, &s, &term);
     }
 
+    /* If there is a tweak then add (or subtract) `msghash` times `tweak` to `s`.*/
+    if (session->pre_session.is_tweaked) {
+        unsigned char msghash[32];
+        secp256k1_scalar e, scalar_tweak;
```

Comment:
I don't mind, but I think everywhere else in this library we avoid defining more than one variable on a line. This is a MISRA thing I think.

Reply:
We don't have that convention, see
 https://github.com/ElementsProject/secp256k1-zkp/blob/secp256k1-zkp/src/ecdsa_impl.h#L185
https://github.com/ElementsProject/secp256k1-zkp/blob/secp256k1-zkp/src/ecdsa_impl.h#L209
for two random examples

Reply:
Oh, ok, I guess I just imagined this and have been adhering to it for no reason.

Carry on.